scripts/steps/launch_database_container.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+echo >&2 ""
+echo >&2 "################################"
+echo >&2 "# Launching database container #"
+echo >&2 "################################"
+echo >&2 ""
+
+docker run \
+--name db \
+-p $PGPORT:$PGPORT \
+--volume $BASE_DIR:/nz-buildings \
+--net $DOCKER_NETWORK \
+-e "PGHOST=$PGHOST" \
+-e "PGPORT=$PGPORT" \
+-e "PGUSER=$PGUSER" \
+-e "PGPASSWORD=$PGPASSWORD" \
+-e "POSTGRES_USER=$PGUSER" \
+-e "POSTGRES_PASSWORD=$PGPASSWORD" \
+-d \
+$DB_DOCKER_IMAGE
+
+RETRIES=3
```

Comment:
would it be worthwhile adding a comment here as to why you are retrying as i thought this is worth knowing and could be a gotcha for folks in the future